deliverables/README.md

```diff
@@ -31,38 +37,98 @@ What is the role of Javascript in a web page?
 
 #### meta
 Why do we separate HTML and CSS?  
+We separate the two so that the pages aren't overly cluttered when editing code. We can easily move from one to the other and edit them easier than if we were to put all of the CSS in-line with the HML.
 
 #### HTML
 What are classes and IDs (and how are they different)?  
+Classes can be used on multiple elements while IDs must contain unique values. IDs are declared in CSS documents using the "#" before the ID name while classes are defined with a "."
+
 What are elements?  
+Elements are what are found in between an opening and closing tag.
+
 What are tags?  
+Tags surround elements and give them meaning.
+
 What are attributes?  
+Attributes are extra information, usually found within an opening tag.
+
 What are forms?  
+Forms are used to collect input from users. From then, the form can manipulate the data according to how it's coded.
+
 What is a div?  
+Divs are placeholders in which users can add properties of CSS to chunks of information.
+
+What are selectors?
+Selectors are the CSS equivalents of tags, which surround properties to give them meaning. 
+
+What are properties? 
+Properties can be given values within selectors to modify a certain CSS styling.
 
-What are selectors?  
-What are properties?  
 What are values?  
+Values are identifiers of properties which can be changed for a desired outcome.
+
 How do CSS styles for a particular element get inherited? ie. how does an element get its "default" styles?  
+There is a hierarchy of properties for every element, the more specific the selector, the higher precedence it takes. Also, the last selector of the same type will take precedence.
+
 What are two CSS attributes you can change to push an element around on the page?  
-What are the three different ways to include CSS in your project or use CSS to style a particular element?  
+You can change "right:" or "left:" in order to move around elements. I was confused by this question since I thought attributes were within HML tags, if it was meant to say properties, then you can use "right:" or "left:" still.
```

Comment:
CSS and HTML can both have attributes. In fact, the style attribute inside and HTML tag allows us to define inline CSS attributes, e.g., `<h1 style="position:relative;">` would define the CSS property of position with the value relative in the style attribute.